developing-plugins/plugin-developers-guide.md

```diff
@@ -15,30 +15,30 @@ The usage of plugin from the user perspective is described [here](/using-plugins
 
 Follow these steps to create a plugin:
 
-1. Use the [GatewayD plugin template for Go](https://github.com/gatewayd-io/plugin-template-go) repository to create a new repository for your plugin.
-2. Clone the repository and start developing your plugin.
-3. Update the `gatewayd_plugins.yml` file with the correct information.
-4. Test your plugin locally using the `make run` target of GatewayD.
-5. Test your plugin in the CI pipeline.
-6. Test your plugin using this [`test.yaml`](https://github.com/gatewayd-io/gatewayd-plugin-cache/blob/main/.github/workflows/test.yaml) workflow.
-7. Publish your plugin to GitHub using this [`release.yaml`](https://github.com/gatewayd-io/gatewayd-plugin-cache/blob/main/.github/workflows/release.yaml) workflow and this [`Makefile`](https://github.com/gatewayd-io/gatewayd-plugin-cache/blob/main/Makefile).
-8. Publish your plugin.
-
-{: .note }
-> You can also use the [GatewayD plugin template for Python](https://github.com/gatewayd-io/plugin-template-python) repository to create a new repository for your plugin. The project is not as mature as the Go template and might have some rough edges.
+1. Generate a plugin scaffold using the `gatewayd plugin scaffold` command.
+2. Update the `gatewayd_plugins.yml` file with the correct information.
```

Comment:
Since the user is developing the plugin, they can just disable checksum verification, hence using `./gatewayd run --dev` or `make run`. I don't see why we should automate this at this point.

Reply:
I will create the ticket for automation, after review 🤞

Reply:
Ticket is created:
https://github.com/gatewayd-io/gatewayd/issues/529